test/e2e/framework/framework.go

```diff
@@ -393,7 +394,14 @@ func (f *Framework) AfterEach() {
 					} else {
 						Logf("Namespace %v was already deleted", ns.Name)
 					}
+				} else {
+					err := WaitForNamespacesDeleted(f.ClientSet, []string{ns.Name}, f.NamespaceDeletionTimeout)
+					if err != nil {
+						Logf("Error waiting for namespace %s to be deleted", ns.Name)
+						nsDeletionErrors[ns.Name] = err
+					}
 				}
+
```

Comment:
Nit: this empty line probably could be removed.

Reply:
same as above. changes into `framework.go` are entirely removed.